src/main/kotlin/graphics/scenery/volumes/HasTransferFunction.kt

```diff
@@ -1,5 +1,10 @@
 package graphics.scenery.volumes
 
+import java.io.BufferedWriter
+import java.io.File
+import java.io.FileWriter
+import java.io.InputStream
+
 /**
  * @author Konrad Michel <Konrad.Michel@mailbox.tu-dresden.de>
```

Comment:
I guess it has more authors now :-)

Reply:
0bacab27966c3598c202ad8f93d6a1e052de3e18 adds author info